<launch>

  <param name="use_sim_time" value="true"/>
  <arg name="stereo_topic" default="stereo_down"/>
  <arg name="image_topic" default="image_rect_color"/>

  <group ns="$(arg stereo_topic)">
    <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc"/>
  </group>
  
  <node pkg="mosaic_pose_extractor" type="mosaic_processor" name="mosaic_processor" output="screen">
    <rosparam file="$(find mosaic_pose_extractor)/default_params.yaml"/>
    <remap from="stereo" to="$(arg stereo_topic)"/>
    <remap from="image" to="$(arg image_topic)"/>
  </node>

  <include file="$(find fugu_configurations)/launch/fugu-c/fugu-c-static_transforms.launch"/>

</launch>
